defaults/os_config_ref.yml

```
@@ -13,7 +13,7 @@ debian:
   buster:
     php_ver: 7.3
```

```suggestion
    php_ver: 8.1
```

I think we can do the same for buster.

---

For `buster` we should also add Debian 10 to this custom repo, because now, I've just added `bullseye` in repo URL. I can change it to be more universal.

---

@wiktor2200 just found this: https://github.com/geerlingguy/ansible-role-php-versions/blob/7ea6036859ba2a36e2bc148258985b94592851d2/tasks/setup-Debian.yml

---

I've seen this role before, but it's whole solution for php and probably could be used in collection. So it's another field for discussion, should we support all edge cases?